components/identifier-generator/back/src/Domain/Model/Property/FreeText.php

```diff
@@ -7,6 +7,7 @@
 use Webmozart\Assert\Assert;
 
 /**
+ * Property for adding free text
```

Comment:
```suggestion
 * Property to add a free text
```
And should we add `to the structure or the identifier` for each Property description? (to explain even better to what it is added)